### PR TITLE
#[150397532] Add travis 150397532

### PR DESCRIPTION
**Pull request by THEozmic**
#### What does this PR do?
Adds Travis and coveralls badges and config

### CODE REVIEW

**Comment by THEozmic** [comment]
I'm merging this PR regardless of the travis result, since travis is the PR. :)

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost:20217/cfh
```
Comment (mradeybee) [inline .env]: Please add this file to your gitignore, you should **NEVER** push sensitive files like this to git.

**Review by mradeybee — CHANGES_REQUESTED** [review]
Remove the `.env` file and add it to your git ignore file. 

Also, add at least one test to make sure that Travis runs successfully.

**Comment by mradeybee** [comment]
Probably you need to wait for @ayeniblessing101's task to be merged.

File: travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+install:
+    - npm install
+script:
+    - npm test
+services:
+  - mongodb
+cache: 
+  - npm install
+notifications:
+  webhooks: https://coveralls.io/webhook?repo_token=AlQ0K4WyO8yno9TSEP9t3SoyrkmoYgI8I
```
Comment (johadi) [inline travis.yml]: I suggest this repo_token should also be included in .env file as it is also a sensitive data

**Review by johadi — COMMENTED** [review]
I also suggest that travis repo_token should be included in the .env file. I see it as a very sensitive data 
`webhooks: https://coveralls.io/webhook?repo_token=AlQ0K4WyO8yno9TSEP9t3SoyrkmoYgI8I`

**Review by mradeybee — CHANGES_REQUESTED** [review]
Set all env variables on travis and not in the code. See how here. https://docs.travis-ci.com/user/environment-variables/

**Review by mradeybee — APPROVED** [review]
GTM

**Pull request merged**